config/configcompression/compressiontype.go

```diff
@@ -5,31 +5,56 @@ package configcompression // import "go.opentelemetry.io/collector/config/config
 
 import "fmt"
 
-type CompressionType string
+// CompressionType represents a compression method
+// Deprecated [0.94.0]: use Type instead.
+type CompressionType = Type
+
+// Type represents a compression method
+type Type string
 
 const (
-	Gzip    CompressionType = "gzip"
-	Zlib    CompressionType = "zlib"
+	TypeGzip    Type = "gzip"
+	TypeZlib    Type = "zlib"
+	TypeDeflate Type = "deflate"
+	TypeSnappy  Type = "snappy"
+	TypeZstd    Type = "zstd"
+	typeNone    Type = "none"
```

Comment:
Should "none" be a valid type?

Reply:
Based on the original PR's conversation there wasn't a public use case for `none`: https://github.com/open-telemetry/opentelemetry-collector/pull/4651#discussion_r780596512.  Until the last 2 weeks there haven't been any material changes to the package so I'm guessing that there still isn't any public use case.